src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navbar: React.FC = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => cn('navbar-item', {
+            'has-background-grey-lighter': isActive,
+          })}
+          to="/"
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          className={({ isActive }) => cn('navbar-item', {
+            'has-background-grey-lighter': isActive,
+          })}
+          to="people"
+        >
```

Comment:
`
<NavbarLink to="/" title="Home" />
`
`
<NavbarLink to="/people" title="People" />
`
Should create custom component for that.